src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
+import { screen, render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('Home', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/Home/i);
   expect(linkElement).toBeInTheDocument();
 });
```

Comment:
- [ ] Please write tests for all components and remember to use jest [snapshots](https://jestjs.io/docs/snapshot-testing)